### PR TITLE
[ID] Add Skin and Skinning

### PR DESCRIPTION
**Pull request by ArdiansyahRZG**
Add Indonesian translation

### CODE REVIEW

File: wiki/Skin/id.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+stub: true
+---
+
+# Skin
+
+*Lihat juga: [membuat skin](/wiki/Skinning)*\
+*untuk sub-forum khusus untuk skin, lihat [membuat skin](https://osu.ppy.sh/community/forums/15)*
```
Comment (Nivalyx) [inline wiki/Skin/id.md:8]: sub-forum names shouldn't be translated as they remain the same no matter what language you're currently browsing it on

image

Comment (ArdiansyahRZG) [reply wiki/Skin/id.md]: Okeyy i already change it

File: wiki/Skinning/id.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Membuat skin
+
+Membuat skin adalah salah satu fitur dari osu!. yang dimana skin tersebut pemain dapat membuat elemen skin tersendiri! skin bisa menjadi beberapa variasi yang dimana bertujuan untuk kesenangan, bertema, ataupun juga dapat menjadi skin yang
```
Comment (hikikuman) [inline wiki/Skinning/id.md:3]:
```suggestion
Membuat skin adalah salah satu fitur dari osu!. Fitur ini digunakan pemain untuk membuat elemen skin tersendiri! Terdapat beberapa variasi dari skin seperti untuk kesenangan, bertemakan sesuatu, ataupun menjadi skin yang
```

File: wiki/Skinning/id.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+*Halaman utama: [Daftar set skin](/wiki/Ranking_Criteria/Skin_Set_List)*
+
+**Hanya untuk beatmap,** jika skin beatmap kamu berisi elemen tunggal daftar yang ditetapkan pada Kriteria Ranking, Ini harus mengandung beberapa elemen yang lainya dalam membuat skin. Ini hanya berlaku untuk skins beatmap, tapi untuk beberapa pembuat skin mungkin dapat mempertimbangkan daftar itu juga.
+                           
+## Daftar elemen skin
+
+List dari beberapa elemen membuat skin yang terbagi menjadi beberapa kategori:
+
+- [Tampilan membuat skin](/wiki/Skinning/Interface)
+- [membuat skin di osu!standard](/wiki/Skinning/osu!)
```
Comment (hikikuman) [inline wiki/Skinning/id.md:17]: These points should be capitalized the same as those contained in en.md

File: wiki/Skinning/id.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+- [membuat skin di osu!taiko](/wiki/Skinning/osu!taiko)
+- [membuat skin di osu!catch](/wiki/Skinning/osu!catch)
+- [membuat skin di osu!mania](/wiki/Skinning/osu!mania)
+- [membuat skin pada suara](/wiki/Skinning/Sounds)
+
+## Lihat juga
+
+- [skin.ini](/wiki/Skinning/skin.ini)
+- [Sejarah membuat skin](/wiki/Skinning/History) - daftar dari skin dengan elemen lama
+- [Ringkasan](https://osu.ppy.sh/community/forums/topics/686664) - daftar dari skin yang telah lengkap
```
Comment (hikikuman) [inline wiki/Skinning/id.md:27]:
```suggestion
- [Ringkasan](https://osu.ppy.sh/community/forums/topics/686664) - daftar dari skin yang telah selesai dibuat
```

File: wiki/Skinning/id.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+- [FAQ membuat skin](/wiki/Skinning/FAQ)
+- [Panduan dan threads penting](/wiki/Skinning/Guides_and_important_threads)
```
Comment (hikikuman) [inline wiki/Skinning/id.md:29]:
```suggestion
- [Panduan dan utas-utas penting](/wiki/Skinning/Guides_and_important_threads)
```

**Review by hikikuman — CHANGES_REQUESTED** [review]

File: wiki/Skinning/id.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+                                                       
+## Skinning sets
+
+*Halaman utama: [Daftar set skin](/wiki/Ranking_Criteria/Skin_Set_List)*
+
+**Hanya untuk beatmap,** jika skin beatmap kamu berisi elemen tunggal daftar yang ditetapkan pada Kriteria Ranking, Ini harus mengandung beberapa elemen yang lainya dalam membuat skin. Ini hanya berlaku untuk skins beatmap, tapi untuk beberapa pembuat skin mungkin dapat mempertimbangkan daftar itu juga.
+                           
+## Daftar elemen skin
+
+List dari beberapa elemen membuat skin yang terbagi menjadi beberapa kategori:
```
Comment (hikikuman) [inline wiki/Skinning/id.md:14]:
```suggestion
Berikut merupakan daftar dari beberapa elemen dalam membuat skin yang terbagi menjadi beberapa kategori:
```
Kata `List` belum diterjemahkan.

File: wiki/Skinning/id.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
+# Membuat skin
+
+Membuat skin adalah salah satu fitur dari osu!. Fitur ini digunakan pemain untuk membuat elemen skin tersendiri! Terdapat beberapa variasi dari skin seperti untuk kesenangan, bertemakan sesuatu, ataupun menjadi skin yang
+hampir tidak mungkin untuk dimainkan.
+                                                       
+## Skinning sets
+
+*Halaman utama: [Daftar set skin](/wiki/Ranking_Criteria/Skin_Set_List)*
+
+**Hanya untuk beatmap,** jika skin beatmap kamu berisi elemen tunggal daftar yang ditetapkan pada Kriteria Ranking, Ini harus mengandung beberapa elemen yang lainya dalam membuat skin. Ini hanya berlaku untuk skins beatmap, tapi untuk beberapa pembuat skin mungkin dapat mempertimbangkan daftar itu juga.
```
Comment (hikikuman) [inline wiki/Skinning/id.md:10]:
```suggestion
**Hanya untuk beatmap,** jika skin beatmap kamu berisi elemen tunggal daftar yang ditetapkan pada Ranking Criteria, Ini harus mengandung beberapa elemen yang lainya dalam membuat skin. Ini hanya berlaku untuk skins beatmap, tapi untuk beberapa pembuat skin mungkin dapat mempertimbangkan daftar itu juga.
```
Berdasarkan diskusi di Discord server Indonesia, tetap menggunakan `Ranking Criteria`.

**Comment by ArdiansyahRZG** [comment]
Reupload files

File: wiki/Skin/id.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
+---
+stub: true
+---
+
+# Skin
+
+*Lihat juga: [membuat skin](/wiki/Skinning)*\
```
Comment (hikikuman) [inline wiki/Skin/id.md:7]:
```suggestion
*Lihat juga: [Membuat skin](/wiki/Skinning)*\
```

File: wiki/Skin/id.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
...
+*Untuk sub-forum khusus tentang skin, lihat [skinning](https://osu.ppy.sh/community/forums/15)*
```
Comment (hikikuman) [inline wiki/Skin/id.md:8]:
```suggestion
*Untuk sub-forum khusus tentang skin, lihat [Skinning](https://osu.ppy.sh/community/forums/15)*
```

File: wiki/Skinning/id.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
+# Membuat skin
+
+Membuat skin adalah salah satu fitur dari osu!. Fitur ini digunakan pemain untuk membuat elemen skin tersendiri! Terdapat beberapa variasi dari skin seperti untuk kesenangan, bertemakan sesuatu, ataupun menjadi skin yang
+hampir tidak mungkin untuk dimainkan.
+                                                       
+## Skinning sets
+
+*Halaman utama: [Daftar set skin](/wiki/Ranking_Criteria/Skin_Set_List)*
```
Comment (hikikuman) [inline wiki/Skinning/id.md:8]:
```suggestion
*Halaman utama: [Daftar Set Skin](/wiki/Ranking_Criteria/Skin_Set_List)*
```

**Review by hikikuman — CHANGES_REQUESTED** [review]

File: wiki/Skinning/id.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
+# Membuat skin
+
+Membuat skin adalah salah satu fitur dari osu!. Fitur ini digunakan pemain untuk membuat elemen skin tersendiri! Terdapat beberapa variasi dari skin seperti untuk kesenangan, bertemakan sesuatu, ataupun menjadi skin yang
+hampir tidak mungkin untuk dimainkan.
+                                                       
+## Skinning sets
+
+*Halaman utama: [Daftar Set Skin](/wiki/Ranking_Criteria/Skin_Set_List)*
+
+**Hanya untuk beatmap,** jika skin beatmap kamu berisi elemen tunggal daftar yang ditetapkan pada Ranking Criteria, Ini harus mengandung beberapa elemen yang lainya dalam membuat skin. Ini hanya berlaku untuk skin beatmap, tapi untuk beberapa pembuat skin mungkin dapat mempertimbangkan daftar itu juga.
```
Comment (hikikuman) [inline wiki/Skinning/id.md:10]:
```suggestion
**Hanya untuk beatmap,** jika skin beatmap kamu berisi elemen tunggal dari daftar set di Ranking Criteria, dan ini harus mengandung beberapa elemen yang lain dalam membuat skin. Hal ini hanya berlaku untuk skin beatmap, tapi untuk beberapa pembuat skin mungkin dapat mempertimbangkan daftar itu juga.
```

**Review by hikikuman — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**